requirements/dev.txt

```diff
@@ -1,9 +1,9 @@
 # Development dependencies.
 black
-dep-license
+#dep-license
 flake8
 isort
 pytest
 twine
-sphinx>=2.0.0
-alabaster
+#sphinx>=2.0.0
+#alabaster
```

Comment:
What's up with these dep changes?

Reply:
I informed you about the issues with venv make previously.  That commit was a mistake. I'll take it out.